src/upload/UploadButton.stories.tsx

```diff
@@ -16,3 +18,23 @@ export const normal = () =>
         }}
         onFilesSelected={action("filesSelected")}
     >Click to upload files</UploadButton>;
+
+normal.story = {
+    parameters: {
+        async puppeteerTest(page: any) {
+            const filesPath = relative(process.cwd(), __dirname);
+            const filesSelectedActionPromise = interceptAction(page, "filesSelected");
+
+            const inputHandle = await page.$("input");
+            await inputHandle.uploadFile(filesPath + "/testFile1", filesPath + "/testFile2");
```

Comment:
Instead of committing these two files to the repository, can we just generate them when we need them, and then destroy them afterwards? I don't like having files lying around in `src/` just for testing this stuff.